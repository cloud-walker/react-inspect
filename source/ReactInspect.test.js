import React from 'react'
import {shallow} from 'enzyme'

import Component from './index'

const subject = shallow(<Component data={{foo: 'bar'}} />)

describe(`${Component.displayName} component`, () => {
  it('should render properly', () => {
    expect(subject).toMatchSnapshot()
  })
})
