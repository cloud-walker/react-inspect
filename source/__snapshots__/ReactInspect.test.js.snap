// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReactInspect component should render properly 1`] = `
<pre>
  {
  "foo": "bar"
}
</pre>
`;
